src/internal/git/push.go

```diff
@@ -18,12 +18,38 @@ const offlineRemoteName = "offline-downstream"
 const onlineRemoteRefPrefix = "refs/remotes/" + onlineRemoteName + "/"
 
 func PushAllDirectories(localPath string) error {
-	// Establish a git tunnel to send the repos
-	tunnel := k8s.NewZarfTunnel()
-	tunnel.Connect(k8s.ZarfGit, false)
-	defer tunnel.Close()
+	gitServerInfo := config.GetGitServerInfo()
+	gitServerURL := ""
+
+	if gitServerInfo.InternalServer {
```

Comment:
It feels like it would be cleaner to just use the gitServerInfo data to build a tunnel if it's in cluster (by detecting if the url ends with `svc.cluster.local`).